### ISSUE TITLE
Set up deployment for cloud environments (manually)

### ISSUE THREAD

**Issue by LindsayYoung** [18F/crt-portal#83]
## Narrative
As a developer, I want to be able to deploy the project's configuration so that we can have three environments. 

## Background
Eventually we will want the following to happen:

1) developer takes ticket and adds feature to the code
2) a separate developer tests the code and reviews it 
3) when they come to an agreement on the PR working it is merged into develop.
4) once the merge occurs and the tests pass, it is autodeployed to the dev environment.
5) we make staging releases every 2 weeks so that the PRs that have been be QAed by relevant SMEs for the release
6) 

(There will also be a design sandbox to make forms and maestro workflows, once designers are satisified, a developer can export that work and start a PR.)

All of that depends on #20 

In the meantime, we can manually push to dev to make sure the codebase is working as we add modules and extend it. 

We want to use staging as the design sandbox. However, in the current deploy setup dev and stage will both be redeployed with new code when there is a change to the master branch on Acquia. 
  
## Acceptance criteria
- [x] we can push to dev without overwriting or otherwise breaking staging

## Technical Notes
- we have gotten dev going manually
- working on getting staging going manually
- need to test that dev deploys won't kill staging
- If that works we should get things going in prod

this will free up #21

**Comment by LindsayYoung** [comment]
- set UUID on staging
- ran updates on staging now /form/bunny- is available 

Tried to deploy to prod and it erased my local directory

**Comment by LindsayYoung** [comment]
Ran into so issues with the deploy with prod documenting them here, we are also still resolving with Acquia support.  https://docs.google.com/document/d/15w2h1R6lm_FbgMdT-f4S4Utir3ev0Ug8QZPXwiMQW4U/edit#

**Comment by LindsayYoung** [comment]
Since the drush update state is manual, pushing to dev doesn't break stage.
